jetty-server/src/main/java/org/eclipse/jetty/server/handler/FileBufferedResponseHandler.java

```diff
@@ -225,6 +220,14 @@ protected void onCompleteFailure(Throwable cause)
                     dispose();
                     callback.failed(cause);
                 }
+
+                @Override
+                public String toString()
+                {
+                    return String.format("FileBufferedInterceptor@%X[fileLength=%d,pos=%d,last=%b]",
+                        this.hashCode(), fileLength, _pos, _last
```

Comment:
Drop `this.`, unnecessary.
Also, field `fileLength` should be prefixed with `_` for consistency.